contrib/templates/scala-job/library/template_variables.tmpl

```diff
@@ -0,0 +1,31 @@
+{{ define `version` -}}
+    0.1
+{{- end }}
+
+{{ define `dbr_version` -}}
+    16.2
+{{- end }}
+
+{{ define `scala_major_minor_version` -}}
+    2.12
+{{- end }}
+
+{{ define `scala_version` -}}
+    {{template `scala_major_minor_version` .}}.18
+{{- end}}
+
+{{ define `java_version` -}}
+    17
+{{- end}}
+
+{{ define `organization` -}}
+    com.examples
```

Comment:
Nit: this should probably just be hardcoded? Since it's also hardcoded in the `src/main.scala/com/examples` path below?